docs/development/prepare-dev-environment.md

```diff
@@ -6,15 +6,15 @@ This guide will provide with detailed instructions on installing all dependencie
 
 Hombrew is a popular package manager for macOS. You can install it by executing the following command in a terminal:
 ```bash
-> /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
Could you also add a couple lines for Linux which suggests using any popular package manager to install such dependencies?

Reply:
The title says that the section is for macOS only.